ttnctl/cmd/clients_edit.go

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var clientEditCmd = &cobra.Command{
+	Use:   "edit [Name]",
+	Short: "Edit the general information of an existing client",
```

Comment:
By "general information", I was thinking of the informative and editable properties of a client (description, URI), by contrast with the un-editable information (such as the secret, the scope, and such).